### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Bethsalazar15**
# Why is this needed?
Because yes cool

## What does this do?
✨Spice ✨

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ Also, one of them had a very, very BIG secret; she could turn into a cupcake if
 Anyways, back to the story…
 They were both going shopping one day then one of them got lost.
 They also forgot where they parked their shopping cart with all of their merchandise in it.
+Then they got kidnapped at the store.
```
Comment (xlgmokha) [inline README.md:10]: Oh no! This is so frightening.

**Comment by AlisaSalikhov** [comment]
I agree...
SPICE

Anyways Its really good!

**Review by CatVidia — APPROVED** [review]
:OOOOOOOOOOOOOOOO

**Review by delilah133 — APPROVED** [review]
SPICYYYY

**Review by GabiCar2 — APPROVED** [review]
oh em gee!!!! conflict!!!!!! angst!!!!!

**Pull request merged**